plugins/modules/ecs_service.py

```diff
@@ -158,6 +158,25 @@
         required: false
         choices: ["EC2", "FARGATE"]
         type: str
+    capacity_provider_strategy:
+        description:
+          - The capacity provider strategy to use with your service.
+        required: false
+        type: list
+        elements: dict
+        suboptions:
+            capacity_provider:
+                description:
+                  - Name of capacity provider.
+                type: str
+            weight:
+                description:
+                  - The relative percentage of the total number of launched tasks that should use the specified provider.
+                type: int
+            base:
+                description:
+                  - How many tasks, at a minimum, should use the specified provider.
+                type: int
```

Comment:
```suggestion
    capacity_provider_strategy:
    version_added: 4.0.0
        description:
          - The capacity provider strategy to use with your service.
        required: false
        type: list
        elements: dict
        suboptions:
            capacity_provider:
                description:
                  - Name of capacity provider.
                type: str
            weight:
                description:
                  - The relative percentage of the total number of launched tasks that should use the specified provider.
                type: int
            base:
                description:
                  - How many tasks, at a minimum, should use the specified provider.
                type: int
```

maybe also add to the description that it is limited to 6 items

> This capacity_provider_strategy is a list of 1-6 dictionaries.